XView: {
  landmarks: {
    type: "SURF"
  },
  matcher: {
    type: "VISUAL"
  }
}

XViewWorker: {
  semantics_image_topic: "/Stereo_Left/Omni_B/labels_image"
}
